qdrant_client/async_qdrant_remote.py

```
@@ -231,35 +241,68 @@ def _parse_url(url: str) -> tuple[Optional[str], str, Optional[int], Optional[st
+                    channel = get_channel(
+                        host=self._host,
+                        port=self._grpc_port,
+                        ssl=self._https,
+                        metadata=self._grpc_headers,
+                        options=self._grpc_options,
+                        compression=self._grpc_compression,
+                        auth_token_provider=self._auth_token_provider,
+                    )
+                    channel_pool.append(channel)
+                self._grpc_channel_pool = channel_pool
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)